src/ir/LocalGraph.cpp

```diff
@@ -61,6 +61,14 @@ struct Info {
   }
 };
 
+// Minimalist block information needed
+struct OptimizedBlock {
+  size_t traverseIdx; // last Traversed Index
+  std::vector<Action> actions; // actions occurring in this block
+  std::vector<OptimizedBlock *> in;
+  std::vector<std::pair<Index, SetLocal*> > lastSets; // for each index, the last set_local for it
```

Comment:
no space between `>>`

Reply:
Ok ... wrong habit from old GCC which dislike this ...